### ISSUE TITLE
Corrupted save when editing anything about the Mii

### ISSUE THREAD

**Issue by Theskrunklydoodle** [Brionjv/Tomodachi-Life-Save-Editor#134]
Everytime I edited anything about a Mii friendship, favorite food,force interaction,etc and when I put the save file into Tomodachi life using checkpoint it says my data was corrupted. I tried to do what the "how to fix" said but it also ended up with corrupted save file I made sure all 3 favorite foods were filled for both miis I changed their favorite foods still corrupted I even tried doing it in a older version and it still got corrupted
So basically if I try to edit anything on a single Mii [even if it's a tiny change like changing their level or hunger] it still corrupted my save
I really hope you can fix this!
And yes the miis only have 1 lover
No extra lover at all
(this is for the non-EX version)

**Comment by Brionjv** [comment]
EX version should fix all of this but it's not ready for now
